node-proxy/openshift-origin-node-proxy.spec

```diff
@@ -132,6 +132,7 @@ fi
 %endif
 %attr(0755,-,-) %{_bindir}/node-find-proxy-route-files
 %attr(0640,-,-) %{_sysconfdir}/openshift/web-proxy-config.json
+%config(noreplace) %{_sysconfdir}/openshift/web-proxy-config.json
```

Comment:
That means that if the administrator has modified `web-proxy-config.json` and upgrades openshift-origin-node-proxy, then `oo-diagnostics` will find the `.rpmnew` file and give a warning.  No big deal, but *might* warrant a brief mention in the doc_text.  Thanks for checking!